ui/src/components/Search.js

```diff
@@ -29,6 +34,9 @@ class Search extends React.Component {
   renderValue = option => {
     // renderValue is used for autocomplete. If I type "foo" into search box,
     // drop-down options whose renderValue contains "foo" will be shown in the drop-down.
+    if (option.value != null) {
```

Comment:
What is the purpose of 37-39? I took it out locally, and things seem to work fine.

Reply:
This will give a javascript warning, you need to look at the console.

Reply:
Ping

Reply:
option.value should be returned as is for chips.